student/forms.py

```diff
@@ -0,0 +1,36 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+from .models import Student, EducationalInstitution, Major
+
+
+class studentRegistrationForm(UserCreationForm):
+    educationalInstitution_Queryset = EducationalInstitution.objects.all()
+    full_name = forms.CharField(max_length=255)
+    email = forms.EmailField(required=True)
+    date_of_birth = forms.DateField()
+    phone_number = forms.CharField(max_length=15)
+    educational_institution = forms.ModelChoiceField(educationalInstitution_Queryset)
+    major = forms.TypedChoiceField(choices=Major.choices)
+    about = forms.CharField(max_length=255)
```

Comment:
255 characters for the about field is too short IMO.
You can change it to: `about = forms.CharField(widget=forms.Textarea)`
This way we can allow a lot of text in this field and the registration form will also indicate it (see attached screenshot).
![image](https://user-images.githubusercontent.com/91069555/145712783-6eae1fe2-5ca3-4125-936b-b8f22c0b2ef3.png)

Reply:
done.